Assets/VRTK/Scripts/Utilities/VRTK_SDKTransformModify.cs

```diff
@@ -0,0 +1,75 @@
+﻿// SDK Transform Modify|Utilities|90064
+namespace VRTK
+{
+    using UnityEngine;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    [Serializable]
+    public class VRTK_SDKTransformModifiers
+    {
+        [Tooltip("An optional SDK Setup to use to determine when to modify the transform.")]
+        public VRTK_SDKSetup sdkType;
```

Comment:
I think this is misleading, since this PR is only doing an equals check with the currently loaded SDK Setup. If we want real "type" checking, we'd probably check the used SDK settings inside the Setup? Though I'm not sure how that should work... And checking for SDK Setup equality is probably fine, since the user can provide their own strange and custom ones.

We just need to make sure users understand how this is used when checking, I'm fine with whatever name, it's just a suggestion from me. 😄